include/ddc/chunk_span.hpp

```diff
@@ -38,14 +38,11 @@ class ChunkSpan<ElementType, DiscreteDomain<DDims...>, LayoutStridedPolicy, Memo
     : public ChunkCommon<ElementType, DiscreteDomain<DDims...>, LayoutStridedPolicy>
 {
 protected:
-    /// the raw mdspan underlying this, with the same indexing (0 might no be dereferenceable)
-    using internal_mdspan_type = std::experimental::mdspan<
-            ElementType,
-            std::experimental::dextents<sizeof...(DDims)>,
-            std::experimental::layout_stride>;
-
     using base_type = ChunkCommon<ElementType, DiscreteDomain<DDims...>, LayoutStridedPolicy>;
 
+    /// the raw mdspan underlying this, with the same indexing (0 might no be dereferenceable)
+    using typename base_type::internal_mdspan_type;
```

Comment:
Was it needed ?

Reply:
Some things didn't compile, not sure this was the fix in the end. In any case I believe this is better than a copy & paste